src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,11 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from "../PostInfo";
+import './PostList.scss';
+
+
+export const PostList = ({ posts }) => (
```

Comment:
The 'posts' prop is not validated. It is a good practice to validate your props to ensure the data your component is receiving is as expected. You can use PropTypes library for prop validation.